src/main/java/com/google/cloud/genomics/dataflow/pipelines/IdentityByState.java

```diff
@@ -89,7 +89,8 @@ public static void validateOptions(Options options) {
 
   }
 
-  private static final String VARIANT_FIELDS = "variants(start,calls(genotype,callSetName))";
+  private static final String VARIANT_FIELDS =
+      "variants(alternateBases,calls(callSetName,genotype),end,referenceBases,start)";
```

Comment:
FYI only. I do worry about the freeform string rep of the requested fields (I know that's not our fault). I'm wondering if there is some best practices / example that we could ask users to do on the API Explorer page to test out the field filter in a lightweight manner?  Just to see if they get back what they were expecting?